### PR TITLE
fix(Tables): Handle Empty Datum

### PR DESCRIPTION
**Pull request by BennettStaley**
Return null for datum with no children
(see faked pagination for an example!)

Fix checkbox header float left on fullWidth

https://docs-icojuhrnkl.now.sh/tables/

### CODE REVIEW

File: pages/tables.js [hunk 1]
```diff
@@ -64,16 +64,16 @@ const data = [
...
+  //   key: '4534r34',
+  //   date: '03/14/15',
+  //   event: 'Really Big Price Change',
+  //   price: '$4,000',
+  // },
 ];
 
 const StyledTable = Table.extend`
-  max-width: 500px;
+  // max-width: 500px;
```
Comment (martahj) [inline pages/tables.js]: [Q] This too

File: pages/tables.js [hunk 2]
```diff
@@ -64,16 +64,16 @@ const data = [
...
-    date: '03/14/15',
-    event: 'Really Big Price Change',
-    price: '$4,000',
-  },
+  // {
+  //   key: '4534r34',
+  //   date: '03/14/15',
+  //   event: 'Really Big Price Change',
+  //   price: '$4,000',
+  // },
```
Comment (martahj) [inline pages/tables.js]: [Q] Any reason this is commented out rather than deleted?

Comment (BennettStaley) [reply pages/tables.js]: This is commenting out the data so that we have an odd number of fields. I'll just delete it instead.

**Comment by BennettStaley** [comment]
⚠️  link uuuupdated!

**Pull request merged**